docs/release-notes.txt

```diff
@@ -3,6 +3,134 @@
 Release Notes
 =============
 
+Version 0.13
+------------
+
+Version 0.13.0
+^^^^^^^^^^^^^^
+
+**Release Date:** August 20, 2020
+
+**New Features**
```

Comment:
Should we include up-top a note that all active experiments must be paused and/or terminated (not sure which) before starting an upgrade?

Reply:
They must be terminated. There's a warning at the top of the migration guide, but two warnings couldn't hurt.